understanding/20/name-role-value.html

```diff
@@ -8,6 +8,15 @@
 <body>
    <h1>Understanding Name, Role, Value</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Authors implement components properly</dd>
```

Comment:
This feels more like an author task than a user-centric objective. Based on the intent section, maybe something like this?

```suggestion
		<dt>Objective</dt><dd>Users of Assistive Technologies (AT) can understand user interface controls</dd>
```

Reply:
(or the same without the "Users of ", even)

Reply:
How about

```suggestion
		<dt>Objective</dt><dd>People using Assistive Technology know the names and states of controls</dd>
```